qai/qconnect/qremoteconnection.py

```diff
@@ -9,10 +9,17 @@ def __init__(
         white_lister=None,
         config_path=["conf", "config.json"],
     ):
-        self.category = category
-
         self.configs = get_configs(config_path=config_path)
 
+        # Only use name passed in as fallback
+        try:
+            self.category = self.configs["SERVICE_NAME"]
+            print("Using SERVICE_NAME from config.json as category")
```

Comment:
don't always print, only print if they set the name in two places
```suggestion
        if category:
            print("Using SERVICE_NAME from config.json as category")
```